Docker File Fails

Still some irregularities in the docker make and I'm getting various errors running $ docker-compose up. 

### 1. Go Server Doesn't Start Due to Linking Issues

`Step 4/23 : RUN CGO_ENABLED=0 GOOS=linux go build -ldflags '-s' -a -installsuffix cgo -v -o server
 ---> Running in a0b17390009e
utils/utils.go:9:2: cannot find package "gopkg.in/mgo.v2" in any of:
	/go/src/github.com/MaximeHeckel/healthDashboard/go/src/server/vendor/gopkg.in/mgo.v2 (vendor tree)
	/usr/local/go/src/gopkg.in/mgo.v2 (from $GOROOT)
	/go/src/gopkg.in/mgo.v2 (from $GOPATH)
apihandlers/apihandlers.go:14:2: cannot find package "gopkg.in/mgo.v2/bson" in any of:
	/go/src/github.com/MaximeHeckel/healthDashboard/go/src/server/vendor/gopkg.in/mgo.v2/bson (vendor tree)
	/usr/local/go/src/gopkg.in/mgo.v2/bson (from $GOROOT)
	/go/src/gopkg.in/mgo.v2/bson (from $GOPATH)
ERROR: Service 'healthdashboard' failed to build: The command '/bin/sh -c CGO_ENABLED=0 GOOS=linux go build -ldflags '-s' -a -installsuffix cgo -v -o server' returned a non-zero code: 1`

### 2. Go Server Name Doesn't Match Github Repo

For example, Dockerfile, go/src/server/main.go and various other files call for github.com/MaximeHeckel/healthDashboard.  I believe this should be updated to match github.com/MaximeHeckel/health-dashboard since govendor pulls there. 

### 3. Directly Running Go Server Fails

Similarly when I run go run main.go inside of /server in the go directory it fails. Compare this with: https://github.com/markwk/apple-health-sync-backend/blob/master/main.go which does work after running govendor for that repo.

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,7 +1,7 @@
 FROM golang:latest as serverbuilder
 
-ADD ./go /go/src/github.com/MaximeHeckel/healthDashboard/go/
-WORKDIR /go/src/github.com/MaximeHeckel/healthDashboard/go/src/server
+ADD ./go /go/src/github.com/MaximeHeckel/health-dashboard/go/
+WORKDIR /go/src/github.com/MaximeHeckel/health-dashboard/go/src/server
 RUN CGO_ENABLED=0 GOOS=linux go build -ldflags '-s' -a -installsuffix cgo -v -o server
 
 ###################################
@@ -25,7 +25,7 @@ RUN yarn build
 
 FROM alpine:latest
 
-COPY --from=serverbuilder /go/src/github.com/MaximeHeckel/healthDashboard/go/src/server/server ./app/
+COPY --from=serverbuilder /go/src/github.com/MaximeHeckel/health-dashboard/go/src/server/server ./app/
 COPY --from=appbuilder /app/build/index.html ./app/
 COPY --from=appbuilder /app/build/manifest.json ./app/
 COPY --from=appbuilder /app/build/asset-manifest.json ./app/

--- a/go/src/server/apihandlers/apihandlers.go
+++ b/go/src/server/apihandlers/apihandlers.go
@@ -7,9 +7,9 @@ import (
 	"net/http"
 	"time"
 
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/auth"
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/health"
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/utils"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/auth"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/health"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/utils"
 	log "github.com/Sirupsen/logrus"
 	"gopkg.in/mgo.v2/bson"
 	"gopkg.in/oleiade/reflections.v1"

--- a/go/src/server/auth/auth.go
+++ b/go/src/server/auth/auth.go
@@ -6,7 +6,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/utils"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/utils"
 )
 
 // CheckAuth basic authentication

--- a/go/src/server/healthgraphql/schema.go
+++ b/go/src/server/healthgraphql/schema.go
@@ -1,8 +1,8 @@
 package healthgraphql
 
 import (
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/apihandlers"
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/health"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/apihandlers"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/health"
 	"github.com/graphql-go/graphql"
 )
 

--- a/go/src/server/main.go
+++ b/go/src/server/main.go
@@ -1,12 +1,12 @@
-package main //import "github.com/MaximeHeckel/healthDashboard/go/src/server"
+package main //import "github.com/MaximeHeckel/health-dashboard/go/src/server"
 
 import (
 	"flag"
 	"net/http"
 
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/apihandlers"
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/healthgraphql"
-	"github.com/MaximeHeckel/healthDashboard/go/src/server/utils"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/apihandlers"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/healthgraphql"
+	"github.com/MaximeHeckel/health-dashboard/go/src/server/utils"
 	log "github.com/Sirupsen/logrus"
 	"github.com/gorilla/mux"
 	"github.com/graphql-go/handler"

